gatorgrade/input/test_in_file_path.py

```diff
@@ -1,17 +1 @@
-'''
-"""Test suite for command_line_generator.py"""
-
-import pytest
-from input import command_line_generator.py
-'''
-
-
-def test_generate_checks_with_file_paths():
-    """Test the checks."""
-
-    # Liam will write a test function here
-    # Given check commands are generated within a file path
-
-    # when generate_checks is run
-
-    # Then file path should be included in the list of strings
+"""Empty file."""
```

Comment:
Is there a reason you need to keep this file around, instead of fully deleting it?

Reply:
I thought it was going to cause a merge conflict, but if that is not the case, then I'll delete it right now.